CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Add missing JSON.stringify in database:settings:set (#3217)
```

Comment:
This is a good description of what the fix was, but can we rephrase this in a way describing what was broken? For example:

```markdown
- Fixes issue where `database:settings:set` would cause an error with valid input.
```